src/operations_protobuf/convert_attest_key.rs

```diff
@@ -129,6 +182,32 @@ mod test {
         let op_attesting_key_name = String::from("attesting key name");
         let op_credential_blob = vec![0xaa; 32];
         let op_secret = vec![0x11; 16];
+        let op_nonce = vec![0x88; 8];
+
+        let proto = OperationProto {
```

Comment:
I would probably put these checks into separate tests, just to make it easier to debug/figure out what's happening if one of them fails. You can probably rename the existing ones something to do with "activate credential"

Reply:
How do names like `attest_key_op_from_proto_CertifyAndQuote` and `attest_key_op_from_proto_ActivateCredential` sound? Or did you have a different idea?

Reply:
That sounds alright!